daffodil-core/src/main/scala/org/apache/daffodil/core/runtime1/ElementBaseRuntime1Mixin.scala

```diff
@@ -24,9 +24,9 @@ import org.apache.daffodil.core.dsom.PrimitiveType
 import org.apache.daffodil.core.dsom.Root
 import org.apache.daffodil.core.dsom.SimpleTypeDefBase
```

Comment:
I wonder if we should modify `setAbsStartBitPos0bInBits` so it does `Assert.invariant(maybeStartPos0bInBits.isEmpty)` and something similar for `setAbsEndBitPos0bInBits`? I think those assertion would have caught this bug since if it fails it means we have set the absolute bit position twice, which we should never do.